js/main.js

```diff
@@ -114,3 +114,14 @@ $feedbackForm.submit( function( e ) {
 
 	e.preventDefault();
 } );
+
+$(function() {
+	// Instant start if input field is filled
+	var $fileForm = $('#file-form'),
+		$fileFormInput = $fileForm.find('#file-form-input'),
+		fileFormVal = $fileFormInput.val();
+
+	if (fileFormVal && fileFormVal !== '') {
+		$fileForm.find('[type="submit"]').trigger('click');
```

Comment:
Because I wasn't sure, whether or not there are any other actions bound to this click-event.